.gitignore

```diff
@@ -16,6 +16,7 @@ packages/tsdocs/fixtures/monorepo/docs
 /docs/site/readmes
 /docs/apidocs/reports-temp
 /docs/apidocs/models
+*.tsbuildinfo
```

Comment:
Let's add `*.tsbuildinfo` to our cli templates for `_.gitigore` too.

Reply:
Added `_.gitignore` entry and a test to verify.